crate/guides/view.md

```diff
@@ -1,37 +1,37 @@
 # View
 
- Visual layout (ie HTML/DOM elements) is described declaratively in Rust, and uses 
-[macros]( https://doc.rust-lang.org/book/appendix-04-macros.html) to simplify syntax. Each element
+ Visual layout (ie HTML/DOM elements) is described declaratively with [macros]( https://doc.rust-lang.org/book/appendix-04-macros.html) to simplify syntax. Each element
 is represented by a macro, eg `div![]`. These act as functions that accept an arbitrary
-number of parameters, in any order: They handle the parameters based exclusively on type.
+number of parameters in any order. They handle parameters based exclusively on type.
 
-The view's defined by a function that's passed to 
-[App::builder](https://docs.rs/seed/0.4.2/seed/struct.App.html#method.build). This takes a `&Model`
-as its parameter, and outputs something that implements the ` View` trait, which is imported in the prelude.
+The view is defined by a function passed to 
+[App::build()](https://docs.rs/seed/0.4.2/seed/struct.App.html#method.build). It takes a `&Model`
+as a parameter and outputs something that implements the ` View` trait, which is imported in the prelude.
 Usually, this is a `Node`, or `Vec<Node>`, representing all nodes that will be inserted as children
-on the top-level one. (The top-level `Node` is in the html file, and specified with
-[App::builder.mount()](https://docs.rs/seed/0.4.2/seed/struct.AppBuilder.html#method.mount), 
-or as a default, the element with id `app`).
+on the top-level one. The top-level `Node` exists in the DOM as specified by
+[AppBuilder::mount()](https://docs.rs/seed/0.4.2/seed/struct.AppBuilder.html#method.mount) and defaults to an element with id `app`.
  It may composed into sub-functions, which can be thought of like components in other frameworks. 
 
 Examples:
 ```rust
 fn view(model: &Model) -> Node<Msg> {
-    h1![ "Let there be light" ],
+    h1![ "Let there be light" ]
```

Comment:
I like trailing commas because it makes row reordering easier and compiler doesn't yell at me when I add another line and forget to add comma after the previous one.

Reply:
Right, but in this case it seems like a return value, so it's a syntactic error, isn't it?

Reply:
Yeah, sorry, I was looking at the other case and I didn't focus enough on the context.